tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -157,7 +157,7 @@ public function testCreateTableWithForeignKeys()
 
         $table = new \Phinx\Db\Table('ntable', array(), $this->adapter);
         $table->addColumn('realname', 'string')
-              ->addColumn('tag_id', 'integer')
+              ->addColumn('tag_id', 'integer', array('signed' => false))
```

Comment:
@robmorgan could use your input on this ^ I'm not really sure what the right answer here is.